src/main/js/html.js

```diff
@@ -80,6 +96,8 @@ export function renderHTML(isd,
     errorHandler,
     previousISDState,
     enableRollUp,
+    options = Object.assign({}, options) || {}, /* https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#deep_clone : */
```

Comment:
`Object.assign({}, options)` will always return an object, so I don't think the `|| {}` is necessary.